### ISSUE TITLE
1.5.4 build failures

### ISSUE THREAD

**Issue by phadej** [ekmett/contravariant#68]
When forcing `transformers-0.6`:

```
                     9.2.0.20210422  9.0.1  8.10.4  8.8.4  8.6.5  8.4.4  8.2.2  8.0.2  7.10.3  7.8.4  7.6.3  7.4.2  7.2.2  7.0.4
contravariant-1.5.4  OK              OK     OK      OK     OK     FAIL   FAIL   FAIL   NO-IP   NO-IP  NO-IP  NO-IP  NO-IP  NO-IP
contravariant-1.5.3  NO-IP           NO-IP  NO-IP   NO-IP  NO-IP  NO-IP  NO-IP  NO-IP  NO-IP   NO-IP  NO-IP  NO-IP  NO-IP  NO-IP
contravariant-1.5.2  NO-IP           NO-IP  NO-IP   NO-IP  NO-IP  NO-IP  NO-IP  NO-IP  NO-IP   NO-IP  NO-IP  NO-IP  NO-IP  NO-IP
contravariant-1.5.1  NO-IP           NO-IP  NO-IP   NO-IP  NO-IP  NO-IP  NO-IP  NO-IP  NO-IP   NO-IP  NO-IP  NO-IP  NO-IP  NO-IP
contravariant-1.5    NO-IP           NO-IP  NO-IP   NO-IP  NO-IP  NO-IP  NO-IP  NO-IP  NO-IP   NO-IP  NO-IP  NO-IP  NO-IP  NO-IP

Build profile: -w ghc-8.4.4 -O0
In order, the following will be built (use -v for more details):
 - contravariant-1.5.4 (lib) (first run)
Configuring library for contravariant-1.5.4..
Preprocessing library for contravariant-1.5.4..
Building library for contravariant-1.5.4..
[1 of 4] Compiling Data.Functor.Contravariant ( old-src/Data/Functor/Contravariant.hs, /codetmp/con/contravariant-1.5.4/.dist-newstyle-trustee/13cc291791f26ca51aed2851dd003eba0257780261849c6556379a682286357e05ec60ce84c38ff6eed02c0d1dfc5ef4c981d84b74e29c70a8ffd4569974e253/build/x86_64-linux/ghc-8.4.4/contravariant-1.5.4/noopt/build/Data/Functor/Contravariant.o )


old-src/Data/Functor/Contravariant.hs:69:1: error:
    Could not find module ‘Control.Monad.Trans.Error’
    Perhaps you meant
      Control.Monad.Trans.Writer (from transformers-0.6.0.2)
      Control.Monad.Trans.Accum (from transformers-0.6.0.2)
      Control.Monad.Trans.Class (from transformers-0.6.0.2)
    Use -v to see a list of the files searched for.
   |
69 | import Control.Monad.Trans.Error
```

EDIT: I think relaxation of https://github.com/ekmett/semigroups/issues/101 would make older GHCs have (failing) installing plans with `contravariant-1.5.4` and `transformers-0.6`, so probably bette to decide what to do with `semigroups` first.

**Comment by RyanGlScott** [comment]
Sigh. I had a feeling that I forgot something. See #69.

**Comment by phadej** [comment]
No worries. I hope I don't find many more `transformers-0.6` issues (I'll try to publish `these`, `indexed-traversable` and `strict` revisions ASAP)

**Comment by RyanGlScott** [comment]
Fixed in 56e52f7. I've uploaded [`contravariant-1.5.5`](https://hackage.haskell.org/package/contravariant-1.5.5) to Hackage with these changes.
